lib/helper/browser.go

```diff
@@ -223,16 +229,39 @@ func OpenBrowserRedirect(target string, typeID uint) error {
 	// generate the federation link
 	federationLink := fmt.Sprintf("%s%s", logoutURL, encodedUrl)
 
-	// open the browser
-	switch runtime.GOOS {
-	case "linux":
-		err = exec.Command("xdg-open", federationLink).Start()
-	case "windows":
-		err = exec.Command("rundll32", "url.dll,FileProtocolHandler", federationLink).Start()
-	case "darwin":
-		err = exec.Command("open", federationLink).Start()
-	default:
-		err = fmt.Errorf("unsupported platform")
+	if config.FirefoxContainers {
+		fmt.Printf("Opening in Firefox Container\n")
```

Comment:
We have a print in `main.go` right before this function is called:
```
fmt.Printf("Federating into %s (%s) via %s\n", favorite.Name, favorite.Account, car.AwsIamRoleName)
```
Thinking we pull that into here and place it under the else condition if the user does not have `config.FirefoxContainirs == true` in order to prevent two similar notifications.

Reply:
I like that, although all the metadata about the session isn't available in the helper, so I added a new struct type to make that cleaner.